tools/interactive/interactivetool_rstudio.xml

```diff
@@ -1,6 +1,6 @@
-<tool id="interactive_tool_rstudio" tool_type="interactive" name="RStudio" version="0.3" profile="22.01">
+<tool id="interactive_tool_rstudio" tool_type="interactive" name="RStudio" version="24.2" profile="22.01">
```

Comment:
Can you make a copy of the new file and add it to `lib/galaxy/config/sample/tool_conf.xml.sample` so we have both versions available ?